aiida_pseudo/cli/__init__.py

```diff
@@ -7,6 +7,6 @@
 click_completion.init()
 
 from .root import cmd_root
-from .install import cmd_install, cmd_install_family, cmd_install_sssp
+from .install import cmd_install, cmd_install_family, cmd_install_sssp, cmd_install_pseudodojo
```

Comment:
Given that pseudo dojo seems to be two words, and you use `PseudoDojo` as capitalization, it would also follow that we use `pseudo_dojo` in filenames

Reply:
This makes sense. I'll go through and change it.